spec/ruby/core/gc/stat_spec.rb

```diff
@@ -1,19 +1,16 @@
 require_relative '../../spec_helper'
 
 describe "GC.stat" do
-  it "supports access by key" do
-    keys = [:heap_free_slots, :total_allocated_objects, :count]
-    keys.each do |key|
-      GC.stat(key).should be_kind_of(Integer)
-    end
-  end
```

Comment:
Should we keep this spec? It should pass now.
Notably it tests that all 3 keys are there.

Reply:
I added three separate specs for the three keys we were tracking. I think separate specs is better, as you can then improve each other time (fore example some might have sensible values.)

Reply:
I see, that's fine now.